.github/workflows/docs.yml

```diff
@@ -0,0 +1,49 @@
+name: Docs
+
+on:
+  release:
+    types: 
+      - created
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Checkout docs
+        uses: actions/checkout@v3
+        with:
+          repository: dappnode/DAppNodeDocs
+
+      - name: Set up dappnodesdk
+        run: npm install -g @dappnode/dappnodesdk@${{ github.event.release.tag_name }}
+
+      - name: Download script from sdk
+        run: wget https://raw.githubusercontent.com/dappnode/DAppNodeSDK/master/.github/workflows/scripts/docs.cjs -O docs.cjs
```

Comment:
This action is weird, you are getting the script from upstream instead of from local